aws-bundle/LICENSE

```diff
@@ -443,6 +443,12 @@ License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
+Group: software.amazon.awssdk  Name: url-connection-client  Version: 2.30.31
```

Comment:
#12299 also added this dependency to for kafka connect module? 

We should update the respective notice and license file in 
a) https://github.com/apache/iceberg/blob/main/kafka-connect/kafka-connect-runtime/main/NOTICE
b) https://github.com/apache/iceberg/blob/main/kafka-connect/kafka-connect-runtime/hive/NOTICE